.github/workflows/_build_torch_with_cuda.yml

```diff
@@ -47,7 +44,7 @@ jobs:
         shell: bash
         run: |
           cd pytorch
-          USE_CUDA=1 python setup.py bdist_wheel
+          TORCH_CUDA_ARCH_LIST="5.2;8.6" USE_CUDA=1 python setup.py bdist_wheel
```

Comment:
I guess `TORCH_CUDA_ARCH_LIST` is needed to make it compile on CPU machine?

Reply:
Yes, this is needed to make it compile on CPU machine